prod/westeurope/support/redis/redis_cache/terragrunt.hcl

```diff
@@ -0,0 +1,66 @@
+dependency "subnet" {
+  config_path = "../subnet"
+}
+
+dependency "storage_account" {
+  config_path = "../storage_account"
+}
+
+dependency "resource_group" {
+  config_path = "../../resource_group"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_redis_cache?ref=v2.1.28"
+}
+
+inputs = {
+  name                  = "backoffice"
+  resource_group_name   = dependency.resource_group.outputs.resource_name
+  capacity              = 1
+  shard_count           = 1
+  enable_non_ssl_port   = false
+  subnet_id             = dependency.subnet.outputs.id
+  family                = "P"
+  sku_name              = "Premium"
+  enable_authentication = true
```

Comment:
@uolter These settings will create a Redis Cluster with shard count of 1? Because in production we will use a `RedisClustr` client ([ref](https://github.com/pagopa/io-functions-backoffice/pull/45/files#diff-bd701b361587be8e65a5fce84cc97955e7fa15953338e72cb4918d220bfcbfe7R11))